nasa/earthdata-search
# EDSC-3936: Updated errors for different shapefile uploads

# Overview

### What is the feature?

Fixed issue where errors wouldn't show up for invalid or bad shapefile uploads.

### What is the Solution?

Added specific error messages for each type of file uploads and make sure they actually show up when those errors are encountered.

### What areas of the application does this impact?

shapefile upload errors

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. go to locahost:8080/search
2. Download all the attachments on the jira https://bugs.earthdata.nasa.gov/browse/EDSC-3963
3. Drag and drop the _Texas.zip_ file onto the map
4. Observe the error "To use a shapefile, please upload a zip file that includes its .shp, .shx, and .dbf files." shows up.
5. Remove the upload using the remove button
6. drag and drop the _.kmz_ file and observe the error "To use a Keyhole Markup Language file, please upload a valid .kml or .kmz file."
7. Remove the upload using the remove button
8. drag and drop the _.json_ file and observe the error "To use a GeoJSON file, please upload a valid .json or .geojson file."
9. Remove the upload using the remove button.
10. Drag and drop the _.rss_ file and observe the error "To use a GeoRSS file, please upload a valid .rss, .georss, or .xml file."

# Checklist
